addon/components/frost-object-browser-filter.js

```diff
@@ -0,0 +1,66 @@
+import Ember from 'ember'
+import layout from '../templates/components/frost-object-browser-filter'
+import _ from 'lodash'
+
+function callIfDefined (context, functionName, ...args) {
```

Comment:
I almost wonder if this concept shouldn't make it's way into Ember core as we seem to do it a lot.